tensorflow/core/kernels/segment_reduction_ops_impl.h

```diff
@@ -1126,6 +1127,94 @@ struct SparseSegmentGradFunctor<CPUDevice, T, Index, SegmentId> {
   }
 };
 
+template <typename T, typename Index, typename SegmentId>
+struct SparseSegmentGradV2Functor<CPUDevice, T, Index, SegmentId> {
+  void operator()(OpKernelContext* context,
+                  SparseSegmentReductionOperation operation,
+                  typename TTypes<T>::ConstMatrix input_flat,
+                  typename TTypes<Index>::ConstVec indices_vec,
+                  typename TTypes<SegmentId>::ConstVec segment_vec,
+                  const TensorShape& dense_output_shape,
+                  typename AsyncOpKernel::DoneCallback done) {
+    (void)done;  // Not used, the CPU implementation is synchronous
```

Comment:
Prefer removing the name of the argument in the function signature.  You can use
```
operator()(..., typename AsyncOpKernel::DoneCallback /*done*/)
```

Reply:
Done